apps/sms/views/conversation/js/compose.js

```diff
@@ -862,7 +919,83 @@ var Compose = (function() {
       };
 
       return defer.promise;
-    }
+    },
+
+    onSegmentInfoChange: function c_onSegmentInfoChange() {
```

Comment:
Honestly when I look at all these notice related methods I feel like we need separate Toaster module, that would look like (simplified):
```
const DEFAULT_HIDE_TIMEOUT = 3000;

exports.Toaster = {
  show: (contentL10n /* type is the same as we use for dialog.show */, options) {
    // dynamically create (clone from template) this simple element
    if (options.persistent) // do not hide this notice
    if (options.timeout) // override default hide timeout
    // otherwise hide using default timeout
   return { hide: () => .... hide(toasterDOMElement)}; 
  }
};
```

I don't really think that _always_ keeping this simple markup in the DOM brings much value or performance win. What do you think about it?